l5kit/l5kit/tests/evaluation/extract_metric_test.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from l5kit.data import ChunkedDataset
+from l5kit.dataset import AgentDataset
+from l5kit.evaluation import compute_error_csv, write_gt_csv, write_pred_csv
+from l5kit.rasterization import StubRasterizer
+
+
+def test_compute_mse_error(tmp_path: Path, zarr_dataset: ChunkedDataset, cfg: dict) -> None:
+    rast = StubRasterizer(
+        (10, 10), np.asarray((0.25, 0.25)), np.asarray((0.5, 0.5)), 0.5
+    )  # TODO replace using builder
```

Comment:
Is this todo fine?

Reply:
I can remove it, replacing with builder actually takes more than writing it like this